build.config

```diff
@@ -98,19 +98,20 @@
       "linux": ["m", "rt"],
       "darwin": [],
       "nuttx": [],
-      "tizen": ["m", "rt"],
+      "tizen": ["m", "rt", "curl"],
       "tizenrt": []
     }
   },
   "module": {
     "always": ["buffer", "console", "events", "fs", "module", "timers"],
-    "include": ["assert", "dns", "http", "net", "stream", "testdriver"],
+    "include": ["assert", "dns", "http", "net", "stream", "testdriver", "https"],
```

Comment:
Having `https` in `include` made Travis CI atleast check if the `https` module builds (it cannot ofcourse run the tests for `Tizen`.)
I will remove it from the `includes` and `excludes` if the above Travis build check for `https` module is not needed.